elcid/templates/partials/lab_test_sparklines.html

```diff
@@ -11,7 +15,18 @@ <h2 class="text-center no-results">Unable to get lab tests for this patient</h2>
     <h2 class="text-center no-results">Failed To load lab tests for this patient</h2>
   </div>
   <div ng-show="patientLoadStatus.isLoaded()">
-    <div ng-hide="data.obs_values.length">
+    {% comment %}
+      we have a result saying that the patients lab tests have loaded but
+      we have not loaded them in yet
```

Comment:
I'm not sure I understand this comment ? You mean 'we think there are tests, this only displays when we haven't fetched them from the API yet' ?

Reply:
Maybe I can explain it better..

Yeah we've validated that tests have loaded, but we have not loaded them in yet.